### PR TITLE
Explain the components

### PR DESCRIPTION
**Pull request by probonopd**
Closes https://github.com/AppImageCommunity/AppImageUpdate/issues/225

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ If you have the optional appimaged daemon installed, then it can use AppImageUpd
 
 ![screenshot from 2016-10-15 16-37-05](https://cloud.githubusercontent.com/assets/2480569/19410850/0390fe9c-92f6-11e6-9882-3ca6d360a190.jpg)
 
+## Components
+
+* `AppImageUpdate`: The GUI application
+* `appimageupdatetool`: A command line tool that does the same but in the terminal
+* `validate`: A tool to check signed AppImages (AppImageUpdate has this built in)
```
Comment (TheAssassin) [inline README.md:29]: "check" can mean anything. You should describe what it really does.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ If you have the optional appimaged daemon installed, then it can use AppImageUpd
 
 ![screenshot from 2016-10-15 16-37-05](https://cloud.githubusercontent.com/assets/2480569/19410850/0390fe9c-92f6-11e6-9882-3ca6d360a190.jpg)
 
+## Components
+
+* `AppImageUpdate`: The GUI application
```
Comment (TheAssassin) [inline README.md:27]:
```suggestion
* `AppImageUpdate`: GUI application to update AppImages with
```

Might seem like a small difference, but you can't repeat yourself too often there.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ If you have the optional appimaged daemon installed, then it can use AppImageUpd
...
+* `appimageupdatetool`: A command line tool that does the same but in the terminal
```
Comment (TheAssassin) [inline README.md:28]:
```suggestion
* `appimageupdatetool`: Commandline application to update AppImages with
```

**Review by TheAssassin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (probonopd) [reply README.md:29]: How about "validates the integrity of the signature built into an AppImage"?

File: README.md (hunk 3 above)
Comment (probonopd) [reply README.md:28]: I think the spelling "command line" is more common, in any case it is what I am using everywhere else.

File: README.md (hunk 1 above)
Comment (TheAssassin) [reply README.md:29]: Sounds good to me. Doesn't make any promises it cannot hold.

File: README.md (hunk 3 above)
Comment (TheAssassin) [reply README.md:28]: True

File: README.md (hunk 1 above)
Comment (probonopd) [reply README.md]: Changed it.

File: README.md (hunk 3 above)
Comment (probonopd) [reply README.md]: Changed it.

**Review by TheAssassin — APPROVED** [review]

**Comment by TheAssassin** [comment]
I'll enable automerge for this repository and squash the PR.

**Pull request merged**